apiserver/offerauth.go

```diff
@@ -72,6 +78,7 @@ func newOfferAuthContext(ctx context.Context, pool *state.StatePool, controllerC
 	if err != nil {
 		return nil, errors.Annotate(err, "unable to get controller config")
 	}
+	modelTag := names.NewModelTag(st.ModelUUID())
```

Comment:
Can we remove this call off of state please ?

Reply:
This one we should talk about, I just wanna make sure I'm passing in the right UUID for the right model